.github/pull-request-template.md

```diff
@@ -0,0 +1,15 @@
+Fixes #
+
+
```

Comment:
this is a place for PR description
often, PR description is self-evident from the issue, thus you don't have to provide it, but here are some reasons why you might want to:

- if you made some changes in the PR that are not directly related to the issue (i.e refactoring or etc)
- if the linked issue had long discussion that reached a conclusion at the end - to save reviewers time and not make them read though the long discussion, PR description could summarize the results of the discussion
- you may document why the code change was implemented a certain way, but ideally this would be documented in the code (i.e why a given library was chosen, or why given solution that on first sight seems non-optimal/overcomplicated is actually the best solution)